third_party/ducc/threading.patch

```diff
@@ -0,0 +1,4 @@
+--- /dev/null	2023-04-06 08:42:12.988021940 +0800
++++ src/ducc0/external/ducc0_custom_lowlevel_threading.h	2023-04-06 14:30:52.458062889 +0800
```

Comment:
Is this necessary?  It should only be included if custom lowlevel threading is defined.

Reply:
Yes, its unnecessary for now, but just cacheing it for multithreaded follow-up PR.